Ham/DabraLibanosHamGG1/msDesc.xml

```diff
@@ -0,0 +1,1782 @@
+                  <handShift cert="high" new="#h3"/>
+                  <seg ana="#invocation" type="interpretation">በአኰቴተ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ </seg>
+                  <seg ana="#provision" type="interpretation">ወሀብኩ፡ አነ፡ ገብረ፡ መስቀል፡ በአሃም፡ ፷፡ ገራህት፡ ለቤተ፡
+                    ክርስቲያን፡ ወከመ፡ ይግ<add>በ</add>ሩ፡ ተዝካሮ፡ ለመጣዕ፡ ከመ፡ ይትወሀሰኒ፡ በመንግሥተ፡ ሰማያት፡ እመቦ፡ ዘተኀየሎ፡
+                    ወዘአገሎ፡ ውጉዘ፡ ለይ<add>ኩ</add>ን። በአፈ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ። <pb/></seg>
+                </q>
+                <q xml:lang="en">
+                  <seg ana="#invocation" type="interpretation"/>In the Name of the Father and the Son
+                  and the Holy Spirit. <seg ana="#provision" type="interpretation">I, Gabra Masqal,
+                    granted sixty fields of ʾAham to the church, so that they celebrate the
+                    commemoration of Maṭāʿ, so that he would beseech a reward for me in the Heavenly
+                    Kingdom. If there is someone who would violate it or defraud it, may he be
+                    excommunicated by the mouth of the Father and the Son and the Holy Spirit.</seg>
+                </q>
+                <note>Translation: I translate ወሀብኩ as 'I granted', while Roger Schneider's translator
+                  offers 'I have given'; ወከመ፡ ይግበሩ፡ ተዝካሮ፡ ለመጣዕ as 'so that they celebrate the
+                  commemoration of Maṭāʿ', while Roger Schneider's translator offers 'so that they may
+                  celebrate the commemoration'; ከመ፡ ይትወሀሰኒ as 'so that he would beseech a reward for me
+                  for me', while Roger Scheider's translator offers 'so that he may be my garantor';
+                  እመቦ፡ ዘተኀየሎ፡ ወዘአገሎ፡ as 'If there is someone who would violate it or defraud it', while
+                  Roger Schneider's translator offers 'whosoever breaks or intringes this'. </note>
+                <note>CCR 4 = [5.0]: (II f. 7v), according to AB. It is page 5 according to Roger
+                  Schneider. Carlo Conti Rossini comments on p. 185 : 'Atto indubbiamente apocrifo, che
+                  trae dal Gadla Libanos.' </note>
+              </item>
+            </list>
+          </additions>
+          
+        </physDesc>
+      </msPart>
+      
+      
+      <msPart xml:id="p6" corresp="#aftertempio">
+<!--        this grouping unit is preserved because described as a unit by CCR-->
+        <msIdentifier>
+          <repository ref="INS0346DL"/>
+        </msIdentifier>
+        <physDesc>
+          <objectDesc>
+           <supportDesc> 
+             <extent>
+              <locus from="19" to="35"/>
+            </extent>
+           </supportDesc>
+          </objectDesc>
+        </physDesc>
+        <!--   f.20 to f.23     -->
+        <msPart xml:id="p7">
+          <msIdentifier>
+            <repository ref="INS0346DL"/>
+          </msIdentifier>
+          <msContents>
+            <msItem xml:id="p7_i3">
+              <locus from="20r2" to="20v12"/>
+<!--              what is on line 1?-->
```

Comment:
locus 20r1-20v12,
the incipit does not include the invocation as typical, but may be this needs to be changed